trustd/build.rs

```diff
@@ -0,0 +1,78 @@
+use std::path::Path;
+use std::process::{Command, ExitStatus};
+use std::{fs, io};
+
+static UI_DIR: &str = "../ui";
+static UI_DIR_SRC: &str = "../ui/src";
+static UI_DIST_DIR: &str = "../ui/client/dist";
+static STATIC_DIR: &str = "../static";
+
+#[cfg(windows)]
+static GIT_CMD: &str = "git.cmd";
+#[cfg(not(windows))]
+static GIT_CMD: &str = "git";
+
+#[cfg(windows)]
+static NPM_CMD: &str = "npm.cmd";
+#[cfg(not(windows))]
+static NPM_CMD: &str = "npm";
+
+fn main() {
+    println!("Build Trustify - build.rs!");
+
+    println!("cargo:rerun-if-changed={}", UI_DIR_SRC);
```

Comment:
```suggestion
    println!("cargo:rerun-if-changed={}", UI_DIR);
```
This is what you meant, I think, and cargo only builds when something beneath there changes.